src/scss/_panktee.scss

```diff
@@ -3,10 +3,14 @@
     margin-bottom: -0.25rem;
 
     &.larivaar {
-      word-break: break-all;
+      // word-break: break-all;
     }
   }
 
+  .wordBreakSpecialChar {
```

Comment:
Can we use some descriptive name? `larivaar-word-breaker` ?

Reply:
Actually its just for one special character where <wbr> (word break html gets break LOL)
Thats why I used wordBreakSPecialChar.
Any suggestion?

Reply:
Maybe you can use inline style for this.

Reply:
DOne